### PR TITLE
Used HTML and CSS to modify the look of our page

### PR DESCRIPTION
**Pull request by sabingaire**
Used HTML and CSS to modify the look of our page.
Now our page looks more colorful. The background has some different colors than white and the message container also has different format and color.
image

### CODE REVIEW

File: src/main/webapp/user-page.html [hunk 1]
```diff
@@ -25,24 +25,32 @@
...
+      <a href="/about.html">About Team 27</a>
     </nav>
-    <h1 id="page-title">User Page</h1>
 
-    <form id="message-form" class="hidden" method="POST" enctype = "multipart/form-data">
-      Enter a new message:
-      <br/>
-      <textarea name="text" id="message-input"></textarea>
-      <br/>
+
```
Comment (elaineliucs) [inline src/main/webapp/user-page.html]: Blank line added unnecessarily. Please remove it!

File: src/main/webapp/css/user-page.css [hunk 2]
```diff
@@ -47,8 +48,22 @@
...
+  border: thin solid black;
+  border-radius: 5px;
+  overflow: hidden;
+}
+
 /* Use this to hide certain elements like forms if the
    user is not logged in or viewing their own page. */
 .hidden {
   display: none;
-}
```
Comment (elaineliucs) [inline src/main/webapp/css/user-page.css:54]: No newline at end of file. I don't think this is a big deal... I believe it's just a message to make it clearer that there isn't one when trying to distinguish between different files. Please correct me if I'm wrong @jonathantanmy .

File: src/main/webapp/css/user-page.css [hunk 3]
```diff
@@ -31,13 +31,14 @@
...
-  padding: 5px;
+  background-image: linear-gradient(to right, white , #ccccff);
 }
 
 /* Give the message body a margin so the text
    isn't too close to the border. */
 .message-body {
-  margin: 5px;
+  margin: 5px
+  font:12pt cursive ;
```
Comment (elaineliucs) [inline src/main/webapp/css/user-page.css]: Be careful with spacing in line 41!

**Comment by elaineliucs** [comment]
Thanks for including an image in your commit message! I would add spacing in between the panels and the navigation of the page so that it looks like the image I have attached. 

Styling

File: src/main/webapp/css/user-page.css (hunk 2 above)
Comment (jonathantanmy) [reply src/main/webapp/css/user-page.css:54]: Thanks for spotting that, Elaine. You're right that this isn't a big deal - Sabin, you don't need to worry about this.

**Comment by jonathantanmy** [comment]
Thanks, Sabin and Elaine. I agree with all of Elaine's suggestions and have nothing to add. :-)

Sabin, once you're done, please let Elaine know. I'll wait until she approves your changes (and clicks the button that says so) before merging.

**Comment by sabingaire** [comment]
image
  Now the page looks like this.

**Review by elaineliucs — APPROVED** [review]

**Pull request merged**